packages/orchestrator-ui-components/src/hooks/useOrchestratorTheme.ts

```diff
@@ -1,18 +1,37 @@
-import { tint, useEuiTheme } from '@elastic/eui';
+import {
+    EuiThemeColorModeStandard,
+    EuiThemeComputed,
+    shade,
+    tint,
+    useEuiTheme,
+} from '@elastic/eui';
 
-export const useOrchestratorTheme = () => {
+import { ColorModes } from '@/types';
+
+export type WfoTheme = {
```

Comment:
You asked for input about naming this type. I dont know a better term, WfoTheme seems fine, it's everything theme related.

Reply:
Yeah might be too close to the `theme` prop we use everywhere. Keeping it like this for now